CDTDatastoreReplicationAcceptanceTests/ReplicationAcceptance.m

```diff
@@ -1879,7 +1879,7 @@ - (void) testRemoteLastSequenceValueAfterPushReplication
     TD_Database *tdb = self.datastore.database;
 
     //see https://github.com/cloudant/CDTDatastore/issues/395
-    [NSThread sleepForTimeInterval:1.0f];
+    [NSThread sleepForTimeInterval:10.0f];
```

Comment:
Given that this is unrelated to the retries and didn't help with the other failure anyway I suggest we drop this commit from the PR and handle it under another issue.